composeApp/src/androidMain/kotlin/org/ooni/engine/AndroidNetworkTypeFinder.kt

```diff
@@ -18,6 +18,9 @@ class AndroidNetworkTypeFinder(
             capabilities.hasTransport(NetworkCapabilities.TRANSPORT_VPN) -> NetworkType.VPN
             capabilities.hasTransport(NetworkCapabilities.TRANSPORT_WIFI) -> NetworkType.Wifi
             capabilities.hasTransport(NetworkCapabilities.TRANSPORT_CELLULAR) -> NetworkType.Mobile
+            capabilities.hasTransport(NetworkCapabilities.TRANSPORT_ETHERNET) -> NetworkType.Unknown("wired_ethernet")
+            capabilities.hasTransport(NetworkCapabilities.TRANSPORT_BLUETOOTH) -> NetworkType.Unknown("bluetooth")
+            capabilities.hasTransport(NetworkCapabilities.TRANSPORT_USB) -> NetworkType.Unknown("usb")
```

Comment:
Why not have them as first class NetworkTypes? The OONI api doesn't accept them?

Reply:
We have to discuss here before I conclude on this.  I did not want to change too much 

We need to also introduce `unknown` which is different from `no_internet`